dev-workflow-ui/src/ch/ivyteam/workflowui/util/ViewerUtil.java

```diff
@@ -35,14 +43,38 @@ public static boolean isViewerAllowed(ICase caze) {
    */
   public static WebLink getViewerLink(IWebStartable startable) {
     if (startable instanceof IProcessWebStartable processStartable) {
-      return ProcessViewer.of(processStartable).url().toWebLink();
+      var urlBuilder = ProcessViewer.of(processStartable).url();
+      var theme = readThemeCookie();
+      System.out.println(theme);
+      urlBuilder.theme(theme);
+      return urlBuilder.toWebLink();
```

Comment:
I would just 
```
var theme = readThemeCookie();
return ProcessViewer.of(processStartable).url().theme(theme).toWebLink()
```

Reply:
Ah yes sure 😄 I forgot to cleanup some old code 😉